src/nile/common.py

```diff
@@ -21,6 +22,8 @@
 TRANSACTION_VERSION = 1
 QUERY_VERSION_BASE = 2**128
 QUERY_VERSION = QUERY_VERSION_BASE + TRANSACTION_VERSION
+ETH_ABI = f"{pt}/artifacts/abis/ERC20.json"
+ETH_ADDRESS = "0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7"
```

Comment:
```suggestion
WRAPPED_ETH_ADDRESS = "0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7"
```

I was confused for the purpose of this ETH_ADDRESS at first glance.

Reply:
what caused the confusion?

I favor ETH over WRAPPED_ETH because:
- you cannot wrap nor unwrap it
- unlike in Ethereum, there's only one of them here in StarkNet
- and the convention seems to be call it Ether:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/1703831/202737072-b2eb10f1-4f2c-44ae-9e56-8bce4e94d6ac.png">

Reply:
WRAPPED was perhaps not the best suggestion (maybe ETHER_ADDRESS? ETH_TOKEN_ADDRESS?). 

The confusion was with the purpose of ETH_ADDRESS (it wasn't clear to me that it was for an ERC20 token, I thought it could have been a bridge or something different until I saw the ABI, keeping in mind that ETH is used for Ethereum a lot, like in `EthAccount.cairo` in cairo-contracts, where is not representing the Ether Token itself). 

But was a small suggestion, ETH_ADDRESS is good enough.

Reply:
ah i get it now. yes it makes sense.

Reply:
Seconded on the initial uncertainty with `ETH_ADDRESS`. I think `ETH_ABI` right next to it implies what it is, but maybe a comment would be helpful. Just a thought